code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -110,7 +110,7 @@
 	return
 
 /obj/item/weapon/gun/projectile/automatic/mini_uzi
-	name = "uzi"
+	name = "\improper 'Type U3' SMG"

```

Comment:
im fairly sure we wont get a C&D over calling one of our guns a uzi.
if you give somebody an item called "uzi", they immediately know what they have in their hands, so I think this should stay.